content/talks.njk

```diff
@@ -30,14 +26,38 @@ summary: |
   slugs=['spot-on', 'popular-tutorial']
 ) }}
 
+{% set fw = collections._talk | findPage('data.feature', 'workshop') %}
+{% if fw %}
+  {{ layout.title('Featured Workshop') }}
+  <article {{ post.aria_label(item) }} class="feature-workshop">
+    {{ post.hero(fw.data.image, 'post') }}
+    <header class="training-header">
+      {{ post.banner(fw) }}
+      {{ post.subtitle(fw.data.sub) }}
+      {{ post.byline(fw, collections) }}
+    </header>
+
+    <div class="training-detail">
+      {{ fw.data.summary | md | safe }}
+
+      {% if fw.data.action %}
+        {{ utility.main_action(
+          fw.data.action.text,
+          url=fw.data.action.url
+        ) }}
+      {% endif %}
+    </div>
+  </article>
+{% endif %}
+
 {{ talks.section(
   title='Upcoming Events',
   events=collections.all | getPublic | getEvents | getFuture,
   all=false,
   by_bird=collections
 ) }}
```

Comment:
@mirisuzanne Did we always show upcoming events like this?  

My suggestions would be to have the name of the event on top, a description (or tickets on sale now) and the presenter's name, then the date/s like the other talk cards. 

Are we showing the presenters sketch here because we don't have an image, or did we always do this?

![Screenshot 2023-10-23 at 3 28 28 PM](https://github.com/oddbird/oddleventy/assets/1581694/af318ec9-a195-47dd-a9bb-ec089e9f5dfa)

Reply:
@stacyk  I didn't make any changes here, this is how we've been displaying future events - grouped by presenter. I think this should be filed as a new issue if we want to revisit that design.

Reply:
@SondraE Would this be one of those shapeshifters?